androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/TrackUtils.kt

```diff
@@ -0,0 +1,99 @@
+package com.profusion.androidenhancedvideoplayer.utils
+
+import androidx.annotation.OptIn
+import androidx.media3.common.C
+import androidx.media3.common.Format
+import androidx.media3.common.TrackSelectionOverride
+import androidx.media3.common.TrackSelectionParameters
+import androidx.media3.common.util.UnstableApi
+import androidx.media3.exoplayer.trackselection.DefaultTrackSelector
+import androidx.media3.exoplayer.trackselection.MappingTrackSelector
+
+const val AUTO_QUALITY_LABEL = "Auto"
+
+@OptIn(UnstableApi::class)
+fun DefaultTrackSelector.generateTrackQualityOptions():
+    ArrayList<Pair<String, TrackSelectionParameters.Builder>> {
```

Comment:
A class to hold the `Pair<String, TrackSelectionParameters.Builder>` it would be better

Reply:
done